kompari/Cargo.toml

```diff
@@ -24,3 +24,8 @@ targets = []
 image = { workspace = true }
 thiserror = { workspace = true }
 walkdir = "2.5"
+oxipng = { workspace = true, optional = true }
```

Comment:
This raises some questions about the exact organisation; my gut reaction is that this image optimisation effectively belongs in `kompari-tasks`.

That is, `kompari` is *only* the things which you need to be included in your test binary.
But getting an initial version of this landed now is good; ultimately I don't really have the bandwidth to contribute heavily at the moment, unfortunately.

Reply:
I suspect that there are more things which should be in the hypothetical `kompari_common`. For example, there's a reasonable argument that even computing the difference images doesn't belong in the Kompari crate.

But I agree that for now, having it here for now is fine.